### ISSUE TITLE
feature request: x & y offsets

### ISSUE THREAD

**Issue by C0R-PSE** [Dr-Turtle/DRG_WeaponHeatCrosshair#4]
some crosshairs (i.e. for drills) are slightly off centered, so an option like this is required to integrate heat indicator in them.

**Comment by Dr-Turtle** [comment]
Not sure if I understand, are you saying the vanilla or modded crosshairs are off? Is this option your proposing just to work around a bug with a game? If so, it's probably not much harder to just fix the big itself

**Comment by Dr-Turtle** [comment]
Didn't mean to close

**Comment by C0R-PSE** [comment]
Some vanilla crosshairs are off centered by developer's intention. What I mean is an option to manage Heat Indicator's position on screen.

**Comment by Dr-Turtle** [comment]
> Some vanilla crosshairs are off centered by developer's intention. What I mean is an option to manage Heat Indicator's position on screen.

As in vertically or horizontally off-center?

**Comment by C0R-PSE** [comment]
Yes, exactly. Won't it be hard to implement? I don't want to bother you.

**Comment by Dr-Turtle** [comment]
> Yes, exactly. Won't it be hard to implement? I don't want to bother you.

ngl, I probably won't get around to fixing this any time soon, there are a lot of other things I want to do first, but haven't had the motivation to work on it lately (as seen in the version/commit history)

**Comment by C0R-PSE** [comment]
Yes, I understand. Thank you for responding!
